src/components/generic/Button.tsx

```diff
@@ -88,7 +88,7 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
     alignItems: 'center',
     padding: 10,
-    height: 80,
+    minHeight: 80,
```

Comment:
@GewoonJaap zou jij op iOS als je alle tekst (settings > accessibility > text size) op z'n grootst zet willen kijken of de knoppen op de pagina's niet aan de onderkant worden afgesneden. Op Android werkt het iig